### PR TITLE
checkout on lmsbranch

### PR DESCRIPTION
**Pull request by vaibhavdeca**
created models but unable to get schema table 
lms1
lms2
lms3
lms4

### CODE REVIEW

File: app/models/teacher.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class Teacher < ApplicationRecord
+    has_many: students
```
Comment (vishalbardewa) [inline app/models/teacher.rb]: Is this valid? based on the ER diagram you submitted

File: app/models/student.rb [hunk 2]
```diff
@@ -1,2 +1,4 @@
 class Student < ApplicationRecord
+    has_many:courses
```
Comment (vishalbardewa) [inline app/models/student.rb:2]: Space

File: db/migrate/20200916094704_create_teachers.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class CreateTeachers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :teachers do |t|
+      t.string :course
+      t.string :name
+      t.string :profile
+
```
Comment (vishalbardewa) [inline db/migrate/20200916094704_create_teachers.rb:7]: add_index in the table

File: app/models/lesson.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+class Lesson < ApplicationRecord
```
Comment (vishalbardewa) [inline app/models/lesson.rb:1]: Relation for Lesson??

File: db/migrate/20200916095119_create_quizzes.rb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+class CreateQuizzes < ActiveRecord::Migration[5.2]
+  def change
+    create_table :quizzes do |t|
+      t.integer :no_of_questions
+      t.integer :total_marks
+      t.integer :obtained_marks
```
Comment (vishalbardewa) [inline db/migrate/20200916095119_create_quizzes.rb:6]: difference between marks_obtained and total_marks

File: app/models/teacher.rb (hunk 1 above)
Comment (vaibhavdeca) [reply app/models/teacher.rb]: I have already updated it today itself, see screenshot just forgot to commit and push done it now. Thanks for pointing out

File: app/models/student.rb (hunk 2 above)
Comment (vaibhavdeca) [reply app/models/student.rb:2]: got it

File: app/models/lesson.rb (hunk 4 above)
Comment (vaibhavdeca) [reply app/models/lesson.rb:1]: actually I have created it in course

File: db/migrate/20200916095119_create_quizzes.rb (hunk 5 above)
Comment (vaibhavdeca) [reply db/migrate/20200916095119_create_quizzes.rb:6]: do I need to find that also?

File: db/migrate/20200916094704_create_teachers.rb (hunk 3 above)
Comment (vaibhavdeca) [reply db/migrate/20200916094704_create_teachers.rb:7]: I did not understand and how to do that

**Pull request merged**